app/views/conversations/new.html.haml

```diff
@@ -1,11 +1,8 @@
+= include_gon camel_case: true
+
 :javascript
   $(document).ready(function () {
-    var data = $.parseJSON( "#{escape_javascript(@contacts_json)}" );
-    new app.views.ConversationsForm({
-      contacts: data,
-      prefillName: "#{h params[:name]}",
-      prefillValue: "#{@contact_ids}"
-    });
+    new app.views.ConversationsForm({prefills: gon.conversationPrefill});
   });
```

Comment:
Can't we try to get rid of this entirely while at it? Shouldn't the backbone router call it?